Formula/openssl@1.1.rb

```diff
@@ -25,8 +25,9 @@ class OpensslAT11 < Formula
 
   on_linux do
     resource "cacert" do
```

Comment:
Let's just make this a separate formula. It's in use in multiple formulae anyway.

Reply:
Not sure of the best strategy for that.

Ideally we'd handle the keychain stuff there and then the file can be used by multiple formula. But the problem is we currently use `openssl` in that processing. A bit of a chicken and egg problem.

Reply:
Or maybe `/usr/bin/openssl` is enough actually. I forget that still exists.

Reply:
I don't follow -- the `cacert` resource is used only in `post_install_from_pem`, where no `openssl` binary is needed. Where do we need `openssl` here?

Or, I guess alternatively we can just use `/etc/ssl/cert.pem`. Is there any reason why we don't use that one?

Reply:
> I don't follow -- the `cacert` resource is used only in `post_install_from_pem`, where no `openssl` binary is needed. Where do we need `openssl` here?

The keychain part. I've just pushed what I was thinking of.

> Or, I guess alternatively we can just use `/etc/ssl/cert.pem`

I can't recall the differences between that and the keychain, but we're going to want to avoid that on older macOS anyway since it's severely outdated.